packages/core/src/components/MonthPicker/MonthPicker.jsx

```diff
@@ -168,14 +175,24 @@ export class MonthPicker extends React.PureComponent {
       this.props.className
     );
     return (
-      <fieldset className={classes}>
-        {this.renderLabel()}
+      <div className={classes}>
+        <div className="ds-c-label">
+          <h4 className="ds-u-margin--0" id={this.labelId}>
```

Comment:
Setting this as an `h4` means it takes on the browser's default styling, like being bold by default. Is there another way to approach this to retain the label styling we use elsewhere? Could we place `ds-c-label` directly on the element instead?